features/bootstrap/IsolatedProcessContext.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+use Behat\Behat\Tester\Exception\PendingException;
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Gherkin\Node\TableNode;
+use Symfony\Component\Process\Process;
+
+/**
+ * Defines application features from the specific context.
+ */
+class IsolatedProcessContext implements Context, SnippetAcceptingContext
+{
+    private $lastOutput;
+
+    /**
+     * @beforeSuite
+     */
+    public static function checkDependencies()
+    {
+        chdir(sys_get_temp_dir());
+        if (!@`which expect`) {
```

Comment:
This method is specifically checking whether expect is present on the system and showing a sensible message if it's not.

IsolatedProcessContext is only used in the 'smoke' suite (not in the `default` suite) so someone checking the project out on Windows and running `bin/behat` won't get any issues